### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by isaacgerg** [tensorflow/tensorboard#1030]
For the scalar plots, can there be an option to mark the min or max of the curves?  Perhaps by putting a box at that point.  It would useful to see where the extreme are and how far your current epoch is from them.

**Comment by chihuahua** [comment]
Hmm, I can see what you mean. I wonder how useful the global min and max are though since they may be outliers. With respect, may I ask why it would be useful to see how far a metric is (at the current epoch) from the min or max?

**Comment by isaacgerg** [comment]
The min AND max together are not useful but either the min or the max is useful depending on what you output for your scalar value.

Here's an example of use, suppose I am looking at a loss curve on a test set and I want to know the exact step in which it overfits.  There is a good chance which the curve will somewhat flatten first before as ascending again.  It would be nice to just have a little dot on the plot showing exactly where that global min occurs.

**Comment by quanvuong** [comment]
+1

**Comment by youyinnn** [comment]
> Hmm, I can see what you mean. I wonder how useful the global min and max are though since they may be outliers. With respect, may I ask why it would be useful to see how far a metric is (at the current epoch) from the min or max?

This statement doesn't make sense. The last result might also be outliers. On the other hand, the min might be the outlier, but that's not the case for max. The maximum result usually comes from well-trained params. It is very important for evaluation.

So I would suggest that marking the max value.
